manifest.json

```
@@ -2146,12 +2146,12 @@
       "id": "terminal",
       "mod_version": "3",
       "remote": "https://github.com/adamharrison/lite-xl-terminal.git:d8ad58405a6c8d82b3ed53b8dd5e80958c36702a",
-      "version": "1.05"
+      "version": "1.05"s
```

You added an `s` by mistake.
```suggestion
      "version": "1.05"
```